doc/theoretical_description_multilabel_classification.rst

```diff
@@ -175,14 +185,17 @@ hypothesis testing. We can express the goal of the procedure as follows:
 In order to find all the parameters :math:`\lambda` that satisfy the above condition, Learn Then Test propose to do the following:
 
 1: First across the collections of functions :math:`(T_\lambda)_{\lambda\in\Lambda}`, we estimate the risk on the calibration data
-\{(x_1, y_1), \dots, (x_n, y_n)\}`.
+:math:`\{(x_1, y_1), \dots, (x_n, y_n)\}`.
+
 2: For each :math:`\lambda_j` in a discrete set :math:`\Lambda = \{\lambda_1, \lambda_2,\dots, \lambda_n\}`, we associate the null hypothesis
-:math:`\mathbb{H}_j: R(\lambda_j)>\alpha`, as rejecting the hypothesis corresponds to selecting :math:`\lambda_j` as a point where risk the risk 
+:math:`\mathbb{H}_j: R(\lambda_j) > \alpha`, as rejecting the hypothesis corresponds to selecting :math:`\lambda_j` as a point where risk the risk 
 is controlled.
-3: For each null hypothesis, we compute a valid p-value using a concentration inequality. Here we choose to compute the Hoeffding-Bentkus p-value
+
+3: For each null hypothesis, we compute a valid p-value using a concentration inequality :math:`p_\lambda_j`. Here we choose to compute the Hoeffding-Bentkus p-value
```

Comment:
There is an error in the run of this code (check build of the documentation).

Reply:
```suggestion
3: For each null hypothesis, we compute a valid p-value using a concentration inequality :math:`p_{\lambda_j}`. Here we choose to compute the Hoeffding-Bentkus p-value
```

Reply:
Done